controllers/storagecluster/noobaa_system_reconciler.go

```diff
@@ -21,6 +21,10 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+const (
+	MonitoringNamespace = "openshift-monitoring"
```

Comment:
@umangachapagain Is it always valid to use this namespace in the core product? I recall that Managed Services prefers to avoid using the default monitoring and wants to have its own. They have the flexibility to deploy it anywhere they want, which is not an issue for us currently because they are not using noobaa. However, if they decide to use it in the future, we will need to modify this. Or shall we do something future-proof now itself?

Reply:
It's ok to add it later. PR looks good to me for current requirements.